common/v2/features/PurchaseMembership/components/MembershipEducation.tsx

```diff
@@ -1,7 +1,205 @@
+import membershipNoAds from 'assets/images/membership/membership-no-ads.svg';
+import membershipNoSponsor from 'assets/images/membership/membership-no-sponsor.svg';
+import membershipStickers from 'assets/images/membership/membership-stickers.svg';
+import membershipShirt from 'assets/images/membership/membership-shirt.svg';
+import { Accordion } from '@mycrypto/ui';
+
+const Heading = styled(FullSizePanelSection)`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+  }
+  display: flex;
+  align-items: flex-start;
+  font-size: 24px;
+  font-weight: bold;
+`;
+
+const Title = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_XS}) {
+    font-size: 24px;
+  }
+  font-size: 32px;
+  font-weight: bold;
+`;
+
+const SImg = styled.img`
+  @media screen and (min-width: ${BREAK_POINTS.SCREEN_MD}) {
+    padding: 0 ${SPACING.XXL};
+  }
+  @media screen and (min-width: ${BREAK_POINTS.SCREEN_SM}) and (max-width: ${BREAK_POINTS.SCREEN_MD}) {
+    padding-right: ${SPACING.LG};
+  }
+
+  width: 50%;
+  height: auto;
+`;
+
+const DescriptionColumn = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_MD}) {
+    margin-top: ${SPACING.BASE};
+  }
+  display: flex;
+  flex-direction: column;
+  & > *:not(:first-child) {
+    margin-top: ${SPACING.LG};
+  }
+`;
+
+const SButton = styled(Button)`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    width: 100%;
+  }
+  width: 250px;
+`;
+
+const SectionTitle = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: ${SPACING.LG};
+  & > :first-child {
+    margin-bottom: ${SPACING.BASE};
+  }
+`;
+
+const ListContainer = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    flex-direction: column;
+    align-items: center;
+  }
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+  margin-bottom: ${SPACING.LG};
+`;
+
+const ListRow = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    width: 90%;
+  }
+  display: flex;
+  flex-direction: column;
+`;
+
+const ListItem = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    width: 100%;
+  }
+  display: flex;
+  align-items: center;
+  min-width: 50%;
+  margin: ${SPACING.SM} 0;
+`;
+
+const ListImg = styled.img`
+  width: 30px;
+  height: auto;
+  margin-right: ${SPACING.XS};
+`;
+
+const PlanContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: ${SPACING.LG};
+`;
+
+const Disclaimer = styled(Typography)`
+  color: ${COLORS.GREY};
+  width: 80%;
+  text-align: center;
+`;
+
+const MembershipEducation = withRouter(({ history }) => {
+  const handleSubmit = () => history.push(`${ROUTE_PATHS.MYC_MEMBERSHIP.path}/buy`);
+  return (
+    <FullSizeContentPanel width={'1100px'}>
```

Comment:
I think this needs to be a styled version of `FullSizeContentPanel` to remove the padding-top, since the header already has bottom-padding. This makes the whitespace in this panel seem excessive.

Figma: 
![image](https://user-images.githubusercontent.com/29407814/78167293-1a1c2f00-7403-11ea-9efa-51aa0d5f7f77.png)

Current Implementation:
![image](https://user-images.githubusercontent.com/29407814/78167321-26a08780-7403-11ea-8424-e1646d4527a1.png)